datasync/downloader/downloader.go

```diff
@@ -827,6 +859,20 @@ func (d *Downloader) fetchHeaders(p *peerConnection, from uint64, pivot uint64)
 			go p.peer.RequestHeadersByNumber(from, MaxHeaderFetch, 0, false)
 		}
 	}
+	getNextPivot := func() {
+		pivoting = true
+		request = time.Now()
+
+		ttl = d.requestTTL()
+		timeout.Reset(ttl)
+
+		d.pivotLock.RLock()
+		pivot := d.pivotHeader.Number.Uint64()
```

Comment:
It could be better to use 'pivotNumber' instead of 'pivot' to increase readability. 'pivot' is used somewhere else for an instance of *types.Header.

Reply:
Updated